contracts/src/v0.8/functions/tests/1_0_0/Setup.t.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {BaseTest} from "./BaseTest.t.sol";
+import {FunctionsRouter} from "../../dev/1_0_0/FunctionsRouter.sol";
+import {FunctionsCoordinator} from "../../dev/1_0_0/FunctionsCoordinator.sol";
+import {FunctionsBilling} from "../../dev/1_0_0/FunctionsBilling.sol";
+import {MockV3Aggregator} from "../../../tests/MockV3Aggregator.sol";
+import {TermsOfServiceAllowList} from "../../dev/1_0_0/accessControl/TermsOfServiceAllowList.sol";
+
+contract FunctionsRouterSetup is BaseTest {
+  FunctionsRouter internal s_functionsRouter;
+  FunctionsCoordinator internal s_functionsCoordinator;
+  MockV3Aggregator internal s_linkEthFeed;
+  TermsOfServiceAllowList internal s_termsOfServiceAllowList;
+
+  uint16 internal s_maxConsumersPerSubscription = 100;
+  uint72 internal s_adminFee = 561724823;
+  bytes4 internal s_handleOracleFulfillmentSelector = 0x0ca76175;
+
+  address internal s_linkToken = 0x01BE23585060835E02B77ef475b0Cc51aA1e0709;
+
+  int256 internal LINK_ETH_RATE = 5021530000000000;
```

Comment:
Why not some nice round numbers for fee and LINK price?

Reply:
Rens probably just pulled the latest feed data. I'll change.